### ISSUE TITLE
onUpdate delta value not updating when chaining

### ISSUE THREAD

**Issue by puresick** [tweenjs/tween.js#465]
tween.js version: 17.4.0.
Browser: tested in Chromium Version 76.0.3809.132 and Firefox 68.0.2

I witnessed an issue when chaining mutliple tweens to repeat themself in a loop like this:

```
const tweenFirst = new TWEEN.Tween({ opacity: 1.0 })
  .to({ opacity: 0.1 }, 2000)
  .onUpdate(delta => {
    console.log(delta.opacity)
  })

const tweenFirstBack = new TWEEN.Tween({ opacity: 0.1 })
  .to({ opacity: 1.0 }, 2000)
  .onUpdate(delta => {
    console.log(delta.opacity)
  })

tweenFirst.chain(tweenFirstBack)
tweenFirstBack.chain(tweenFirst)

tweenFirst.start()
```

Looping both tweens like this will result in `delta` only showing start and end value without the values inbetween after both tweens run at least once. The first run of either tween shows the correct delta values.

**Comment by mikebolt** [comment]
I know what's wrong and I know how to fix it.

Until I resolve this, I suggest chaining tweens by calling `start()` in onComplete callbacks.

**Comment by mikebolt** [comment]
Hi. I fixed this issue in v18.2. The library was inferring the start values whenever the tween started, even when it started the tween as a result of a chain. Now it will only infer the start values when they are unset, which will probably be when the tween first starts. I added a unit test and I also tested it manually with the code snippet you provided. Reopen the issue if it still occurs.

**Comment by puresick** [comment]
Thank you, that was fast!
